test/test-usage.js

```diff
@@ -1,68 +1,52 @@
-var assert = require('assert');
-var async = require('async');
+'use strict';
+
+var concat = require('concat-stream');
 var debug = require('debug')('strong-pm:test');
+var executor = require.resolve('../bin/sl-pm.js');
+var fork = require('child_process').fork;
```

Comment:
execFile can leak sl-pm, but fork establishes an IPC channel, so when test-usage dies, it can't leak its children.